package.json

```diff
@@ -26,39 +26,47 @@
 		"polyfill"
 	],
 	"dependencies": {
-		"assert": "^2.0.0",
+		"assert": "^2.1.0",
 		"browserify-zlib": "^0.2.0",
 		"buffer": "^6.0.3",
 		"console-browserify": "^1.2.0",
 		"constants-browserify": "^1.0.0",
 		"crypto-browserify": "^3.12.0",
 		"domain-browser": "^4.22.0",
 		"events": "^3.3.0",
-		"filter-obj": "^2.0.2",
 		"https-browserify": "^1.0.0",
 		"os-browserify": "^0.3.0",
 		"path-browserify": "^1.0.1",
 		"process": "^0.11.10",
-		"punycode": "^2.1.1",
+		"punycode": "^2.3.0",
 		"querystring-es3": "^0.2.1",
-		"readable-stream": "^4.0.0",
+		"readable-stream": "^4.4.2",
 		"stream-browserify": "^3.0.0",
 		"stream-http": "^3.2.0",
 		"string_decoder": "^1.3.0",
 		"timers-browserify": "^2.0.12",
 		"tty-browserify": "^0.0.1",
-		"type-fest": "^2.14.0",
-		"url": "^0.11.0",
-		"util": "^0.12.4",
+		"type-fest": "^4.4.0",
+		"url": "^0.11.3",
+		"util": "^0.12.5",
 		"vm-browserify": "^1.1.2"
 	},
 	"devDependencies": {
-		"ava": "^3.15.0",
+		"ava": "^5.3.1",
 		"p-webpack": "^1.0.1",
-		"webpack": "^5.73.0",
-		"xo": "^0.39.1"
+		"webpack": "^5.88.2",
+		"xo": "^0.56.0"
 	},
 	"peerDependencies": {
 		"webpack": ">=5"
+	},
+	"xo": {
+		"rules": {
+			"unicorn/prefer-module": "off",
+			"comma-dangle": "off"
+		},
+		"ignores": [
+			"fixture.js"
```

Comment:
Can we remove thus?

Reply:
We have these for a 4 line file 🤷 Not worth it imo.
<img width="893" alt="image" src="https://github.com/Richienb/node-polyfill-webpack-plugin/assets/19473034/93ee3a9e-0755-4c6b-9c1e-84e10c9de464">

Reply:
Add esline rule ignores in a comment at the top of the file in a single line.

Reply:
Also, if you follow the suggested fixes, it breaks the test.

Reply:
Yeah don't apply the fixes, they should instead be ignored at the top of the file

Reply:
Done ✅